packages/console/src/pages/Applications/components/CreateForm/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { ApplicationType } from '@logto/schemas';
+import React, { SVGProps } from 'react';
+import { useController, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import FormField from '@/components/FormField';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import { applicationTypeI18nKey } from '@/types/applications';
+
+import TypeDescription from '../TypeDescription';
+import * as styles from './index.module.scss';
+
+const Close = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
Maybe we could move all icons into a separate place, for we may use them anywhere. eg. `Close`, `Copy`

Reply:
it's more like a "business component" here, but you are right, it fits to all modals. will update soon